airbyte-webapp/src/pages/DestinationPage/pages/CreateDestinationPage/components/DestinationForm.tsx

```diff
@@ -65,6 +68,7 @@ export const DestinationForm: React.FC<DestinationFormProps> = ({
       actionDescription: "Destination connector type selected",
       connector_destination: connector?.name,
       connector_destination_definition_id: destinationDefinitionId,
+      ...(trackParams && { ...trackParams }),
```

Comment:
```suggestion
      ...trackParams,
```

You can spread `undefined` into an object, it will not add anything in that case, thus no need to check if the object existed here.